homeassistant/components/linkplay/media_player.py

```diff
@@ -290,6 +293,73 @@ async def async_play_preset(self, preset_number: int) -> None:
         """Play preset number."""
         await self._bridge.player.play_preset(preset_number)
 
+    @exception_wrap
+    async def async_join_players(self, group_members: list[str]) -> None:
+        """Join `group_members` as a player group with the current player."""
+
+        controller: LinkPlayController = self.hass.data[DOMAIN][CONTROLLER]
+        multiroom = self._bridge.multiroom
+        if multiroom is None:
+            multiroom = LinkPlayMultiroom(self._bridge)
+
+        for group_member in group_members:
+            bridge = self._get_linkplay_bridge(group_member)
+            if bridge:
+                await multiroom.add_follower(bridge)
+
+        await controller.discover_multirooms()
+
+    def _get_linkplay_bridge(self, entity_id: str) -> LinkPlayBridge:
+        """Get linkplay bridge from entity_id."""
+
+        entity_registry = er.async_get(self.hass)
+
+        # Check for valid linkplay media_player entity
+        entity_entry = entity_registry.async_get(entity_id)
+
+        if (
+            entity_entry is None
+            or entity_entry.domain != Platform.MEDIA_PLAYER
+            or entity_entry.platform != DOMAIN
+            or entity_entry.config_entry_id is None
```

Comment:
Are we sure media player doesn't already guard for this?

Reply:
This code was taken from a different integration, the media_player integration doesn't guard against this.